src/polycubed/src/extiface.cpp

```diff
@@ -372,6 +373,8 @@ std::string ExtIface::get_parameter(const std::string &param_name) {
       // netdev does not have an Ip
       return "";
     }
+  } else if (param_upper == PARAMETER_NAME) {
```

Comment:
I think it is better than removing the API from the transparent_cube, but please remember to change the caller to get_parent_parameter(PARAMETER_NAME).